src/sagemaker/jumpstart/cache.py

```diff
@@ -218,8 +219,7 @@ def _get_manifest_key_from_model_id_semantic_version(
 
         error_msg = f"Unable to find model manifest for '{model_id}' with version '{version}'. "
         error_msg += (
-            "Visit https://sagemaker.readthedocs.io/en/stable/doc_utils/pretrainedmodels.html"
-            " for updated list of models. "
+            f"Visit {MODEL_ID_LIST_WEB_URL} for updated list of models. "
```

Comment:
Could you add a line explaining that customers should upgrade to the latest version of the SDK to access all the models?